persistent-qq/test/Spec.hs

```diff
@@ -19,33 +19,27 @@ import PersistTestPetType
 import PersistentTestModels
 
 main :: IO ()
-main = hspec specs
+main = hspec spec
 
 _debugOn :: Bool
 _debugOn = False
 
-sqlite_database_file :: Text
-sqlite_database_file = "testdb.sqlite3"
-sqlite_database :: SqliteConnectionInfo
-sqlite_database = mkSqliteConnectionInfo sqlite_database_file
-
 runConn :: MonadUnliftIO m => SqlPersistT (LoggingT m) t -> m ()
 runConn f = do
-  let debugPrint = _debugOn
-  let printDebug = if debugPrint then print . fromLogStr else void . return
+  let printDebug = when _debugOn . B.putStrLn . fromLogStr
   flip runLoggingT (\_ _ _ s -> printDebug s) $ do
-    _ <- withSqlitePoolInfo sqlite_database 1 $ runSqlPool f
+    _ <- withSqliteConn ":memory:" $ runSqlConn f
```

Comment:
What motivated this change?

Reply:
@parsonsmatt creating a file in the source tree interferes with file watching (specifically, `hspec/sensei`).  If you don't like the in-memory approach, I can create the sqlite-file somewhere under `/tmp` instead.

Reply:
It's totally fine, I'm just curious for the change. In memory seems like it'd be preferable anyway, and I'm not sure why it was setup like that.